package.json

```diff
@@ -83,6 +83,7 @@
     "eslint": "^8.8.0",
     "eslint-plugin-notice": "^0.9.10",
     "eslint-plugin-react-hooks": "^4.3.0",
+    "find-up": "^6.3.0",
```

Comment:
It looks like this is your prod dependency, not dev. It would need to go to packages/playwright-test/bundles/utils/package.json. Better yet implement it in fileUtils, it is a two-liner.